src/pug/templates/index.pug

```diff
@@ -0,0 +1,9 @@
+doctype html
+html(lang='en-US')
+    head
+        meta(charset='UTF-8')
+        title
+            Chrome | console.redhat.com
+        include ../head.pug
+    body
+        include ../body.pug
```

Comment:
Once this takes an effect, we would only really need a simple `<div id="foo" />` and the chrome script tags. Everything else should be then injected as a React component.

Reply:
Yup, I kept this in here just for the simplicity.